tests/components/automation/test_init.py

```diff
@@ -162,18 +162,20 @@ async def test_trigger_service_ignoring_condition(hass, calls):
                 "alias": "test",
                 "trigger": [{"platform": "event", "event_type": "test_event"}],
                 "condition": {
-                    "condition": "state",
+                    "condition": "numeric_state",
                     "entity_id": "non.existing",
-                    "state": "beer",
+                    "above": "1",
                 },
                 "action": {"service": "test.automation"},
             }
         },
     )
 
-    hass.bus.async_fire("test_event")
-    await hass.async_block_till_done()
-    assert len(calls) == 0
+    with patch("homeassistant.components.automation.LOGGER.warning") as logwarn:
```

Comment:
We have access to the `caplog` fixture to assert logging.

https://docs.pytest.org/en/stable/logging.html#caplog-fixture

Reply:
Thanks, I will look into that.

Reply:
#46214